homeassistant/components/govee_light_local/light.py

```diff
@@ -70,16 +76,27 @@ def __init__(
 
         capabilities = device.capabilities
         color_modes = {ColorMode.ONOFF}
+
         if capabilities:
-            if GoveeLightCapability.COLOR_RGB in capabilities:
+            if GoveeLightFeatures.COLOR_RGB & capabilities.features:
                 color_modes.add(ColorMode.RGB)
-            if GoveeLightCapability.COLOR_KELVIN_TEMPERATURE in capabilities:
+            if GoveeLightFeatures.COLOR_KELVIN_TEMPERATURE & capabilities.features:
                 color_modes.add(ColorMode.COLOR_TEMP)
                 self._attr_max_color_temp_kelvin = 9000
                 self._attr_min_color_temp_kelvin = 2000
-            if GoveeLightCapability.BRIGHTNESS in capabilities:
+            if GoveeLightFeatures.BRIGHTNESS & capabilities.features:
                 color_modes.add(ColorMode.BRIGHTNESS)
 
+            if (
+                GoveeLightFeatures.SCENES & capabilities.features
+                and capabilities.scenes
+            ):
+                self._attr_supported_features |= LightEntityFeature.EFFECT
+                self._attr_effect_list = [_NONE_SCENE.capitalize()]
+                self._attr_effect_list.extend(
+                    s.capitalize() for s in capabilities.scenes
+                )
```

Comment:
Is this list static? Let's add translations for it, so ideally all the values should be snake_case